tests/Avalonia.UnitTests/MockPlatformRenderInterface.cs

```diff
@@ -22,6 +22,21 @@ public class MockPlatformRenderInterface : IPlatformRenderInterface
             return Mock.Of<IFormattedTextImpl>();
         }
 
+        public IEllipseGeometryImpl CreateEllipseGeometry(Rect rect)
+        {
```

Comment:
Could these methods also return a Mock?

Reply:
Took a while - surely it can return mocks - I kept original implementation as I was not sure if it needs to. Adding it shouldn't cause any issues.